adyenv6core/src/com/adyen/v6/commands/AdyenCaptureCommand.java

```diff
@@ -141,14 +142,18 @@ private boolean supportsManualCapture(String paymentMethod) {
             case "klarna":
```

Comment:
```suggestion
            case "klarna":
            case "afterpay":
            case "ratepay":
```
adding a few openinvoice methods as defined here: https://github.com/Adyen/adyen-magento2/blob/5dfbe4d2033bc88d86b8dc676aa80aae3e6eb851/Model/Cron.php#L1642-L1645